Gemfile.lock

```diff
@@ -553,7 +553,7 @@ DEPENDENCIES
   rspec-example_disabler!
   rspec-legacy_formatters
   rspec-rails (~> 3.2.0)
-  rubocop (~> 0.28)
+  rubocop (~> 0.30)
```

Comment:
This `0.30` looks bogus.

As soon as I run `bundle` my `Gemfile.lock` is updated and replaces the `0.30` with `0.32`...